index.html

```diff
@@ -1282,7 +1282,9 @@ <h3>Contexts</h3>
 meaning of the terms that are used by their application. One mechanism for
 doing so is described in the Section on
 <a data-cite="VC-DATA-INTEGRITY#validating-contexts">Validating Contexts</a> in
-the [[[VC-DATA-INTEGRITY]]] specification.
+the [[[VC-DATA-INTEGRITY]]] specification. Specification authors MAY require
+that JSON-LD Contexts be integrity protected by using the `relatedResource`
+feature described in Section [[[#integrity-of-related-resources]]].
```

Comment:
I think what this is about is ecosystems that want to provide their own special profiling rules (but also look below at the other option):

```suggestion
the [[[VC-DATA-INTEGRITY]]] specification. Profiles MAY require
that JSON-LD contexts be integrity protected by using the `relatedResource`
feature described in Section [[[#integrity-of-related-resources]]].
```

Reply:
Alternatively, for language that is more verbose but more similar to what is in the controller document:

```suggestion
the [[[VC-DATA-INTEGRITY]]] specification. Other specifications that profile
this specification MAY require that JSON-LD contexts be integrity protected
by using the `relatedResource` feature described in Section
[[[#integrity-of-related-resources]]].
```